test/e2e/framework/framework.go

```diff
@@ -0,0 +1,103 @@
+package framework
+
+import (
+	"crypto/tls"
+	"flag"
+	"fmt"
+	"net/http"
+	"path/filepath"
+
+	"github.com/yahoo/athenz/clients/go/zms"
+	athenzClientset "github.com/yahoo/k8s-athenz-syncer/pkg/client/clientset/versioned"
+	"github.com/yahoo/k8s-athenz-syncer/pkg/log"
+	"github.com/yahoo/k8s-athenz-syncer/pkg/util"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+type Framework struct {
+	K8sClient           kubernetes.Interface
+	ZMSClient           zms.ZMSClient
+	AthenzDomainsClient athenzClientset.Clientset
+}
+
+var Global *Framework
+
+// Setup() create necessary clients for tests
+func setup() error {
+	// config
+	var kubeconfig *string
+	if home := util.HomeDir(); home != "" {
+		kubeconfig = flag.String("kubeconfig", filepath.Join(home, ".kube", "config"), "(optional) absolute path to the kubeconfig file")
+	} else {
+		kubeconfig = flag.String("kubeconfig", "", "absolute path to the kubeconfig file")
+	}
+	inClusterConfig := flag.Bool("inClusterConfig", true, "Set to true to use in cluster config.")
+	key := flag.String("key", "/var/run/athenz/service.key.pem", "Athenz private key file")
+	cert := flag.String("cert", "/var/run/athenz/service.cert.pem", "Athenz certificate file")
+	zmsURL := flag.String("zms-url", "", "Athenz ZMS API URL")
+	logLoc := flag.String("log-location", "/var/log/k8s-athenz-syncer/k8s-athenz-syncer.log", "log location")
+	logMode := flag.String("log-mode", "info", "logger mode")
+	flag.Parse()
+
+	// init logger
+	log.InitLogger(*logLoc, *logMode)
+
+	if *inClusterConfig {
```

Comment:
if `kubeconfig` path is passed in, would the default `inClusterConfig: true` override it?